bin/express

```diff
@@ -60,6 +61,39 @@ if (!exit.exited) {
 }
 
 /**
+ * Replace placeholders for serving index page
+ */
+
+function addIndexServing(app, viewEngine) {
+  if (program.view == 'none') {
+    app.locals.requireIndexRoute = '';
+    app.locals.indexRoute = '';
+  } else {
+    app.locals.requireIndexRoute = eol + "var index = require('./routes/index');";
+    app.locals.indexRoute = eol + "app.use('/', index);";
+  }
+}
+
+/**
+ * Replace the template placeholders according to the configured view engine
+ */
+
+function addViewEngine(app, viewEngine) {
+  if (program.view == 'none') {
+    app.locals.viewEngineSetUp = '';
+    app.locals.renderError = "res.sendFile('error.html', {\n" +
```

Comment:
Above you're using the `eol` variable for the new line, but here just `\n`. Use one or the other?